back/db/migrate/20210430154637_add_confirmation_required_field_to_users.user_confirmation.rb

```diff
@@ -0,0 +1,16 @@
+class AddConfirmationRequiredFieldToUsers < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :confirmation_required, :boolean, default: true, null: false
+
+    active_users = User.active
+    users_that_do_not_require_confirmation = active_users.reject(&:should_require_confirmation?)
```

Comment:
I think processing all users of all platforms in ruby might be too intensive for a migration and potentially break the platforms. Would it be possible to find a similar solution that does this in one or just a few SQL queries? Otherwise, maybe best check with Koen if this is acceptable.

We won't be able to notice if an update fails.

Reply:
Yes it should be possible, will give it a try